css/style.css

```diff
@@ -75,6 +75,189 @@ a.logo:hover {
   margin-left: 22px;
 }
 
+.works-container {
+  align-items: center;
+  padding: 20px;
+  display: grid;
+  background-color: #fff;
+}
```

Comment:
- Kindly add the margin-top to your card container so that your design will stick to the Figma design as I show it below. You must stick to the design as much as possible (e.g., font, colors, images, text, margins) using [the templates in Figma](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=0%3A1). :+1: :grinning: :grinning: :grinning: 
 
## Your design 
![image](https://user-images.githubusercontent.com/95297251/191938320-c859a171-7e70-4a34-a919-a7dee74620d9.png)

## Figma design 
![image](https://user-images.githubusercontent.com/95297251/191938714-1de0f6df-abc1-4f4f-bfc5-78eaa7df48d7.png)